docker/runtime/nodejs-6.10/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:6.10-alpine
```

Comment:
we should be using our bitnami images and avoid alpine.
we can ask sameer and use the prod node container he is working on